updated disclaimer, archive detection, clean up flags

JeBobs opened the pull request:
No more beeps, no more persistent warnings, no more insults, just a single silent disclaimer that also helps the user get started using their own games.

Updated the archived format detection to add tar/gz, and updated the box to state that xenia does not support running archived games.

Restored iso path logging from 71b8c2357e56d5dd7aca064cb8ef54771639b6fc

illusionyy commented on third_party/capstone:
Was this intentional

JeBobs replied:
no, I'll remove it

illusionyy commented on src/xenia/vfs/devices/disc_image_device.cc:
```diff
@@ -30,10 +30,10 @@ DiscImageDevice::~DiscImageDevice() = default;
     return false;
   } else {
-    XELOGFS("DiscImageDevice::Initialize");
+    XELOGFS("DiscImageDevice::Initialize({})", xe::path_to_utf8(host_path_));
```
Someone will whine about this change set

JeBobs replied:
I can change it back, was just going to revert it to how it was pre-message

illusionyy replied:
Personally I don't particularly care. If people do witch hunt then that's their problem. We'll probably catch some rare path bug with it.

JeBobs committed:
updated disclaimer, archive detection, clean up flags

No more beeps, no more persistent warnings, no more insults, just a single silent disclaimer that also helps the user get started using games from their console.

Updated the archived format detection to add tar/gz, and updated the box to state that xenia does not support running archived games.

JeBobs commented:
should be all fixed up, if there's any other changes I'm happy to update it
